k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -260,6 +257,11 @@ public void onCryptoModeChanged(CryptoMode cryptoMode) {
         recipientPresenter.onCryptoModeChanged(cryptoMode);
     }
 
+    @Override
+    public void onOpenPgpInlineChange(boolean enabled) {
+        recipientPresenter.onCryptoPgpInlineChanged(enabled);
+    }
+
     enum Action {
```

Comment:
Correct, but there isn't really a much nicer way for getting a callback from the PgpInlineDialog through to the presenter other than having the activity implement an interface. It's also consistent with the way `onCryptoModeChanged` is handled.